src/ResponseTagger/Delegator/XLocationIdBCTagger.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license   For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\PlatformHttpCacheBundle\ResponseTagger\Delegator;
+
+use EzSystems\PlatformHttpCacheBundle\ResponseConfigurator\ResponseCacheConfigurator;
+use EzSystems\PlatformHttpCacheBundle\ResponseTagger\ResponseTagger;
+use Symfony\Component\HttpFoundation\Response;
+
+class XLocationIdBCTagger implements ResponseTagger
```

Comment:
Could you add a summary of what the Tagger does, and in what context it is (not) useful ?

Reply:
done!